src/plugins/cse/mod.rs

```diff
@@ -0,0 +1,265 @@
+                })
+            })
+            .filter_map(|result: Result<CseMetadata, String>| {
+                // 경고 메시지 모아서 출력
+                // 메타데이터 크롤링 실패 시 크롤링 대상에서 제외
+                result
+                    .inspect_err(|msg| warn!("[{}] {}", T::IDENTIFIER, msg))
+                    .ok()
+            })
+            .collect::<Vec<CseMetadata>>();
+
+        Ok(posts_metadata)
+    }
+
+    async fn fetch_post(&self, metadata: &CseMetadata) -> Result<SsufidPost, PluginError> {
+        let html = reqwest::get(&metadata.url)
+            .await
+            .map_err(|e| PluginError::request::<T>(e.to_string()))?
+            .text()
+            .await
+            .map_err(|e| PluginError::parse::<T>(e.to_string()))?;
+
+        let document = Html::parse_document(&html);
+
+        let title = document
+            .select(&self.selectors.title)
+            .next()
+            .map(|span| span.text().collect::<String>().trim().to_string())
+            .ok_or_else(|| PluginError::parse::<T>("Title element not found".to_string()))?;
+
+        let thumbnail = document
+            .select(&self.selectors.thumbnail)
+            .next()
+            .and_then(|img| img.value().attr("src"))
+            .unwrap_or_default()
+            .to_string();
+
+        let content = document
+            .select(&self.selectors.content)
+            .next()
+            .ok_or_else(|| PluginError::parse::<T>("Content element not found".to_string()))?
+            .child_elements()
+            .map(|p| p.text().collect::<String>().replace('\u{a0}', " ")) // &nbsp 제거
+            .collect::<Vec<String>>()
+            .join("\n");
+
+        let attachments = document
+            .select(&self.selectors.attachments)
+            .filter_map(|a| a.value().attr("href"))
+            .map(|s| s.to_string())
+            .collect();
+
+        Ok(SsufidPost {
+            id: metadata.id.clone(),
+            url: metadata.url.clone(),
+            author: metadata.author.clone(),
+            title,
+            category: vec![],
+            created_at: metadata.created_at,
+            updated_at: None,
+            thumbnail,
+            content,
+            attachments,
+        })
+    }
+}
+
+pub mod bachelor;
+pub mod employment;
+pub mod graduate;
+
+#[cfg(test)]
+mod tests {
+    use crate::plugins::cse::bachelor::CseBachelorPlugin;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn test_crawler_fetch_metadata() {
+        let crawler: CseCrawler<CseBachelorPlugin> = CseCrawler::new();
+
+        // 1 페이지의 게시글 메타데이터 목록 가져오기
+        let metadata_list = crawler.fetch_metadata(1).await.unwrap();
+        assert!(!metadata_list.is_empty());
+
+        for metadata in &metadata_list {
+            println!("{:?}", metadata);
+        }
+
+        let first_metadata = &metadata_list[0];
+        assert!(!first_metadata.id.is_empty());
+        // assert!(!first_metadata.url.trim().starts_with("https"));
```

Comment:
```suggestion
    assert!(first_metadata.url.trim().starts_with("https"));
```
로 수정하면 테스트 통과할 것 같아요!

Reply:
아 ㅋㅋ 바보였네요...